scripts/btgrep.py

```diff
@@ -5,10 +5,13 @@
 import sys
 import urllib
 import urllib2
+import base64
 
-# You need to fill in the domain of where you install
-# Botanist for this to work, obviously :)
-BOTANIST_DOMAIN = 'http://example.com'
+BOTANIST_DOMAIN = 'https://codesearcher.int.sproutsocial.com/'
```

Comment:
It may be worth force pushing to remove that internal domain from this public pull request - not that it's accessible outside your network, but if someone gets on the network, they'd know where to go